daemon/SecretsImpl/secrets.cpp

```diff
@@ -2586,3 +2594,23 @@ void Daemon::ApiImpl::SecretsRequestQueue::handleFinishedRequest(
     }
 }
 
+void Daemon::ApiImpl::SecretsRequestQueue::dealWithData() const
```

Comment:
Didn't try to compile, but why the method is called dealWithData() here, while everywhere you call it with dealWithDataCorruption() (which sounds a better name).

Reply:
Just a typo, will fix.